src/pages/Cart/Cart.js

```diff
@@ -1,8 +1,184 @@
+  //     .then(result => setCartList(result));
+  // }, []);
+
+  //백엔드 통신
+  const getCartList = () => {
+    fetch('http://10.58.52.172:3000/carts/', {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization:
+          'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJkYXRhIjoxMiwiaWF0IjoxNjY5MDI2ODA5LCJleHAiOjE2NzE2MTg4MDksImlzcyI6ImFkbWluIiwic3ViIjoiYWNjZXNzVG9rZW4ifQ.DhfgeERBkf4s7uin2NCCSLX2tFNcWXRs-vgMvY4InJs',
+      },
+    })
+      .then(response => response.json())
+      .then(result => setCartList(result)); //콜백함수//
+  };
+
+  useEffect(() => {
+    getCartList();
+  }, []);
+
+  //총 가격 계산
+
+  const totalPrice = cartList
+    .filter(el => checkItems.includes(el.productOptionId))
+    .reduce((a, b) => a + b.quantity * b.price, 0);
+  //삭제 버튼
+
+  //삭제 버튼 통신
+  const cartDelete = productOptionId => {
+    // setCartList(
+    //   cartList.filter(product => product.productOptionId !== productOptionId)
+    // );
+    fetch(
+      `http://10.58.52.172:3000/carts/delete?product_option_id=${productOptionId}`,
+      {
+        method: 'DELETE',
+        headers: {
+          'Content-Type': 'application/json',
+          Authorization:
+            'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJkYXRhIjoxMiwiaWF0IjoxNjY5MDI2ODA5LCJleHAiOjE2NzE2MTg4MDksImlzcyI6ImFkbWluIiwic3ViIjoiYWNjZXNzVG9rZW4ifQ.DhfgeERBkf4s7uin2NCCSLX2tFNcWXRs-vgMvY4InJs',
+        },
+      }
+    )
+      .then(response => response.json())
+      .then(result => {
+        if (result.message === 'product deleted') {
+          alert('삭제가 완료되었습니다.');
+          getCartList();
+        } else {
+          alert('다시 시도해주세요!');
+        }
+      });
+  };
+
+  // if (!cartList.productOptionId) return null;
+  // fetch('http://10.58.52.56:3000/carts/', {
+  //   method: 'GET',
+  //   headers: {
+  //     'Content-Type': 'application/json',
+  //     Authorization:
+  //       'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJkYXRhIjoxMiwiaWF0IjoxNjY5MDI2ODA5LCJleHAiOjE2NzE2MTg4MDksImlzcyI6ImFkbWluIiwic3ViIjoiYWNjZXNzVG9rZW4ifQ.DhfgeERBkf4s7uin2NCCSLX2tFNcWXRs-vgMvY4InJs',
+  //   },
+  // })
+  //   .then(response => response.json())
+  //   .then(result => setCartList(result));
+
+  // useEffect(() => {
+  //   fetch('http://10.58.52.56:3000/carts/', {
+  //     method: 'GET',
+  //     headers: {
+  //       'Content-Type': 'application/json',
+  //       user_id: 8,
+  //     },
+  //   })
+  //     .then(response => response.json())
+  //     .then(result => setCartList(result)); //콜백함수//
+  // }, []);
+
+  return (
+    <div className="container">
+      <div className="cart">
+        <div className="cartContainer">
+          <div className="titleContainer">
+            <input
+              className="allCheckBox check"
+              type="checkbox"
+              onChange={e => handleAllCheck(e.target.checked)}
+              checked={checkItems.length === cartList.length ? true : false}
```

Comment:
```suggestion
              checked={checkItems.length === cartList.length}
```

해당 조건의 결과값 자체가 boolean 값이기 때문에 삼항연산자를 사용하지 않아도 됩니다!